ReactAndroid/src/main/java/com/facebook/react/uimanager/JSTouchDispatcher.java

```diff
@@ -116,14 +116,21 @@ public void handleTouchEvent(MotionEvent ev, EventDispatcher eventDispatcher) {
       mGestureStartTime = TouchEvent.UNSET;
     } else if (action == MotionEvent.ACTION_MOVE) {
       // Update pointer position for current gesture
+      float[] moveTargetCoordinates = new float[2];
```

Comment:
Should we update `mTargetCoordinates` instead? If we do so, `UP` event also will have proper coordinates, right?